File: app/src/main/java/com/ladsoft/bakingapp/data/network/api/BakingAppApi.java
package com.ladsoft.bakingapp.data.network.api;

import com.ladsoft.bakingapp.data.network.entity.RecipePayload;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface BakingAppApi {
    @GET("/android-baking-app-json")
    Call<List<RecipePayload>> recipes();
}

File: app/src/main/java/com/ladsoft/bakingapp/data/network/api/BakingAppApi.kt
package com.ladsoft.bakingapp.data.network.api


import com.ladsoft.bakingapp.data.network.entity.RecipePayload
import retrofit2.Call
import retrofit2.http.GET

interface BakingAppApi {

    @GET("/android-baking-app-json")
    fun recipes(): Call<List<RecipePayload>>
}
